Tests/unit/New-PolarisStaticRoute.Tests.ps1

```diff
@@ -50,7 +53,12 @@
         #  Confirm no other routes created
         $AllRoutes = @()
         $AllRoutes += Get-PolarisRoute
-        $AllRoutes.Count | Should Be 1
+        $AllRoutes.Count | Should Be 2
+    }
+
+    It "Should create a static route a default file (index.html in this case)" {
+        $Route = Get-PolarisRoute -Path /BaseRoot 
+        $Route | Should not BeNullOrEmpty
```

Comment:
Instead of checking for an additional route can we add an e2e test that actually calls the root and verifies the content of that file matches the index.html?